### ISSUE TITLE
what to do with `\cite{RFC7230}`?

### ISSUE THREAD

**Issue by simsong** [iluxonchik/rfc-bibtex#10]
The `datatracker.ietf.org` service returns RFC BiBTeX entries with the letters `rfc` in lowercase:

```bibtex
@misc{rfc7230,
	series =	{Request for Comments},
	number =	7230,
	howpublished =	{RFC 7230},
	publisher =	{RFC Editor},
	doi =		{10.17487/RFC7230},
	url =		{https://rfc-editor.org/rfc/rfc7230.txt},
        author =	{Roy T. Fielding and Julian Reschke},
	title =		{{Hypertext Transfer Protocol (HTTP/1.1): Message Syntax and Routing}},
	pagetotal =	89,
	year =		2014,
	month =		jun,
	abstract =	{The Hypertext Transfer Protocol (HTTP) is a stateless application-level protocol for distributed, collaborative, hypertext information systems. This document provides an overview of HTTP architecture and its associated terminology, defines the "http" and "https" Uniform Resource Identifier (URI) schemes, defines the HTTP/1.1 message syntax and parsing requirements, and describes related security concerns for implementations.},
}
```

How should we handle the situation when a person does a `\cite{RFC7230}` ? I think that BibTeX labels are case-sensitive. Should we modify the code that we receive from `ietf.org`, or should we not accept `\cite{RFC7230}`, only `\cite{rfc7230}`, or should we just leave it as is, and have `\cite{RFC7230}` bring in a BiBTeX entry, and then ignore it?

**Comment by iluxonchik** [comment]
@simsong apologies for inactivity, I have been extremely busy with work. I will answer to this and the remaining issues very soon.

**Comment by simsong** [comment]
No problem, we are all crazy busy.

**Comment by iluxonchik** [comment]
@simsong I think that we should accept both version `\cite{RFC123}` and `\cite{rfc123}`.

This gives the flexibility to the user (some prefer lowercase, some uppercase). There is no obvious reason why someone would want to have `RFC123` and `rfc123` point to two different things.

**Comment by simsong** [comment]
So we will need to preserve case in the .bib file.

**Comment by iluxonchik** [comment]
Yes. What are your thoughts on this? Do you agree?

**Comment by simsong** [comment]
I think that preserving case is good. But what do we do if a sloppy person has both \cite{rfc7230} and \cite{RFC7230} ? Putting it twice in the .bib file will result in duplicate entries, which will be confusing. So do we throw an error?

**Comment by iluxonchik** [comment]
You are raising a very interesting question. I suggest showing a warning message for now.

**Comment by iluxonchik** [comment]
Closed in #12
